.circleci/config.yml

```diff
@@ -436,15 +436,15 @@ _environments:
 executors:
   debug_exec:
     docker:
-      - image: rosplanning/navigation2:main
+      - image: ghcr.io/ros-planning/navigation2:main
     working_directory: /opt/overlay_ws
     environment:
       <<: *common_environment
       CACHE_NONCE: "Debug"
       OVERLAY_MIXINS: "debug ccache coverage-gcc"
   release_exec:
     docker:
-      - image: rosplanning/navigation2:main
+      - image: ghcr.io/ros-planning/navigation2:main
```

Comment:
Should we do something about our existing dockerhub? Update links in the readme to the new docker image registry? Update the badge for the pull count?

Reply:
How would a user find these images? I don't really see a way through the github interface on the nav2 landing page, I think we'd have to add a link (And probably do a discourse blast that we're moving entirely from dockerhub)

Reply:
> Should we do something about our existing dockerhub? Update links in the readme to the new docker image registry?

We should remove existing tags from our repo on the dockerhub registry to avoid any pulling of stale images. We can also modify the repo description to denote it's deprecation and redirect traffic to the new github image registry.

> Update the badge for the pull count?

Not sure there is a badge for that yet with github container registry, though the pull count does is displayed from container package listing on the ros-planning org.

https://shields.io/

> How would a user find these images? I don't really see a way through the github interface on the nav2 landing page, I think we'd have to add a link (And probably do a discourse blast that we're moving entirely from dockerhub)

Yeah, I kind of wish that the tags for container registry where showcased under the `Packages` sidebar. This seems to be the case for my personal fork, but not for organization repos yet. Perhaps that may change after the github beta.

The image tag only includes an environment to build nav2 packages from source and not prebuilt nav2 binaries, so I'm not sure who else would be pulling them other than our CI and fellow maintainers. But we could do the discourse blast if you'd prefer.

Reply:
OK, please update the readme with links to the registry prominently placed and a note somewhere below about moving from dockerhub. We can remove tags form dockerhub / ockerhub badge